Tools/CISupport/build-webkit-org/factories_unittest.py

```diff
@@ -940,7 +940,7 @@ class TestExpectedBuildSteps(unittest.TestCase):
             'delete-stale-build-files',
             'compile-webkit'
         ],
-        'Apple-tvOS-Simulator-15-Release-Build': [
+        'Apple-tvOS-Simulator-16-Release-Build': [
```

Comment:
Find and replace went too far, don't think you intended to switch this yet.

Reply:
Indeed. The unit tests caught me on this as well.